components/Forms/ComponentFactory.tsx

```diff
@@ -172,8 +178,14 @@ export const ComponentFactory: React.VFC<FactoryProps> = observer(
                         values={
                           field.type == 'boolean'
                             ? [
-                                { key: 'true', text: 'Yes' },
-                                { key: 'false', text: 'No' },
+                                {
+                                  key: 'true',
+                                  text: router.locale == 'en' ? 'Yes' : 'Oui',
+                                },
+                                {
+                                  key: 'false',
+                                  text: router.locale == 'en' ? 'No' : 'Non',
```

Comment:
shouldn't yes/no/oui/non be in the translations file as well?

Reply:
100%!